### PR TITLE
Add routine to calculate blackbody spectrum

### PR DESCRIPTION
**Pull request by jschwab**

### CODE REVIEW

File: wsynphot/spectrum1d.py [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+    dlambda : ~float
+        Wavelength interval for spectrum.
+
+
+    Returns
+    -------
+    bb : `~starkit.fix_spectrum1d.SKSpectrum1D`
+        Blackbody spectrum.
+    """
+
```
Comment (wkerzendorf) [inline wsynphot/spectrum1d.py]: `temperature = u.Quantity(temperature, u.K)`

File: wsynphot/spectrum1d.py [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+
+def blackbody1d(temperature, radius, distance=10*u.pc,
+                lambda_min=2000, lambda_max=10000, dlambda=1):
+    """
+    One dimensional blackbody spectrum.
+
+    Parameters
+    ----------
+    temperature : `~astropy.units.Quantity`
```
Comment (wkerzendorf) [inline wsynphot/spectrum1d.py]:
```suggestion
    temperature : astropy.units.Quantity or float (IF FLOAT Kelvin is assumed!!)
    
```

File: wsynphot/spectrum1d.py [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+
+
+    Returns
+    -------
+    bb : `~starkit.fix_spectrum1d.SKSpectrum1D`
+        Blackbody spectrum.
+    """
+
+    wavelength = np.arange(lambda_min, lambda_max, dlambda) * u.AA
+
```
Comment (wkerzendorf) [inline wsynphot/spectrum1d.py]:
```suggestion
if not hasattr(radius, 'unit'):
    raise ValueError("radius needs to be a quantity (e.g. 1 * u.cm)")
```

File: wsynphot/spectrum1d.py (hunk 1 above)
Comment (jschwab) [reply wsynphot/spectrum1d.py]: This is done in blackbody_lambda

**Comment by wkerzendorf** [comment]
@jschwab I can't merge this because it's a draft - can you convert it.

**Comment by jschwab** [comment]
I hadn't converted it because I haven't actually tested it in my own application yet.  I probably won't get to that until next week.  But I went ahead and made it ready to review.  Take your pick to merge or wait.

**Comment by jschwab** [comment]
This seems to be working as expected, so I think you're good to merge.

**Pull request merged**